src/data/balance/signature-species.ts

```diff
@@ -11,87 +11,87 @@ export type SignatureSpecies = {
-  RAMOS: [Species.SKIDDO, Species.HOPPIP, Species.BELLSPROUT],
-  CLEMONT: [Species.HELIOPTILE, Species.MAGNEMITE, Species.EMOLGA],
-  VALERIE: [Species.SYLVEON, Species.MAWILE, Species.MR_MIME],
-  OLYMPIA: [Species.ESPURR, Species.SIGILYPH, Species.SLOWKING],
-  WULFRIC: [Species.BERGMITE, Species.SNOVER, Species.CRYOGONAL],
+  VIOLA: [Species.SCATTERBUG, Species.SURSKIT, Species.CUTIEFLY, Species.DWEBBLE],
+  GRANT: [Species.TYRUNT, Species.AMAURA, Species.BINACLE, Species.HISUI_AVALUGG],
+  KORRINA: [Species.LUCARIO, Species.MIENFOO, Species.HAWLUCHA, Species.PANCHAM],
+  RAMOS: [Species.SKIDDO, Species.HOPPIP, Species.BELLSPROUT, [Species.PHANTUMP, Species.PUMPKABOO]],
+  CLEMONT: [Species.HELIOPTILE, Species.MAGNEMITE, Species.DEDENNE, Species.SHINX],
+  VALERIE: [Species.SYLVEON, Species.MAWILE, Species.MR_MIME, [Species.SPRITZEE, Species.SWIRLIX]],
+  OLYMPIA: [Species.ESPURR, Species.SIGILYPH, Species.INKAY, Species.SLOWKING],
+  WULFRIC: [Species.BERGMITE, Species.SNOVER, Species.CRYOGONAL, Species.AMAURA],
   // Gym Leaders- Galar
-  MILO: [Species.GOSSIFLEUR, Species.APPLIN, Species.BOUNSWEET],
-  NESSA: [Species.CHEWTLE, Species.ARROKUDA, Species.WIMPOD],
-  KABU: [Species.SIZZLIPEDE, Species.VULPIX, Species.TORKOAL],
-  BEA: [Species.GALAR_FARFETCHD, Species.MACHOP, Species.CLOBBOPUS],
-  ALLISTER: [Species.GALAR_YAMASK, Species.GALAR_CORSOLA, Species.GASTLY],
-  OPAL: [Species.MILCERY, Species.TOGETIC, Species.GALAR_WEEZING],
-  BEDE: [Species.HATENNA, Species.GALAR_PONYTA, Species.GARDEVOIR],
-  GORDIE: [Species.ROLYCOLY, Species.STONJOURNER, Species.BINACLE],
-  MELONY: [Species.SNOM, Species.GALAR_DARUMAKA, Species.GALAR_MR_MIME],
-  PIERS: [Species.GALAR_ZIGZAGOON, Species.SCRAGGY, Species.INKAY],
-  MARNIE: [Species.IMPIDIMP, Species.PURRLOIN, Species.MORPEKO],
-  RAIHAN: [Species.DURALUDON, Species.TURTONATOR, Species.GOOMY],
+  MILO: [Species.GOSSIFLEUR, Species.SEEDOT, Species.APPLIN, Species.LOTAD],
+  NESSA: [Species.CHEWTLE, Species.WIMPOD, Species.ARROKUDA, Species.MAREANIE],
+  KABU: [Species.SIZZLIPEDE, Species.VULPIX, Species.GROWLITHE, Species.TORKOAL],
+  BEA: [Species.MACHOP, Species.GALAR_FARFETCHD, Species.CLOBBOPUS, Species.FALINKS],
+  ALLISTER: [Species.GASTLY, Species.GALAR_YAMASK, Species.GALAR_CORSOLA, Species.SINISTEA],
+  OPAL: [Species.MILCERY, Species.GALAR_WEEZING, Species.TOGEPI, Species.MAWILE],
+  BEDE: [Species.HATENNA, Species.GALAR_PONYTA, Species.GARDEVOIR, Species.SYLVEON],
+  GORDIE: [Species.ROLYCOLY, [Species.SHUCKLE, Species.BINACLE], Species.STONJOURNER, Species.LARVITAR],
+  MELONY: [Species.LAPRAS, Species.SNOM, Species.EISCUE, [Species.GALAR_DARUMAKA, Species.GALAR_MR_MIME]],
+  PIERS: [Species.GALAR_ZIGZAGOON, Species.SCRAGGY, Species.TOXEL, Species.INKAY], // Tera Dark Toxel
+  MARNIE: [Species.IMPIDIMP, Species.MORPEKO, Species.PURRLOIN, Species.CROAGUNK], // Tera Dark Croagunk
+  RAIHAN: [Species.DURALUDON, Species.TRAPINCH, Species.GOOMY, Species.TURTONATOR],
```

Comment:
```suggestion
  RAIHAN: [Species.DURALUDON, Species.TRAPINCH, Species.TURTONATOR, Species.SANDACONDA], // Tera Dragon Sandaconda
```
Similar to his gym team in SwSh, he could use Sandaconda to set up sand, and use the tera without losing Steel typing on Duraludon/Arch

Reply:
His strategy revolved around having a sand setter on a team consisting of Pokemon that can stick around in sand. He drops the sand gimmick altogether in consecutive fights in order to add things like Goodra/Turtonator and this would ring true with random spawns as they would also be negatively impacted by Sand. This is also primarily a doubles centric strategy and since Raihan is currently a singles only gym leader (would hope he is made a double battle when that system gets fine tuned) it would not be as beneficial to him in the few scenarios it would otherwise.